web-common/src/layout/BlockingOverlayContainer.svelte

```diff
@@ -32,7 +32,10 @@
         style:font-size="48px"
       >
         <div class="on" style="--length: {2000 + Math.random() * 5000}ms;">
-          <Spinner {status} duration={300 + Math.random() * 200} />
+          <DelayedSpinner
```

Comment:
Does it make sense here? If a blocking overlay is present it should always have a spinner no?

Reply:
https://github.com/rilldata/rill/pull/5351/commits/e352da7bde1f34e6c06100ed614ac52892aa069c